packages/react-flicking/src/demo/plugins/Arrow.tsx

```diff
@@ -0,0 +1,28 @@
+import * as React from "react";
+import Flicking from "../../react-flicking/Flicking";
+import { Arrow } from "@egjs/flicking-plugins";
+import "../css/plugins.css";
+
+export default class ArrowDemo extends React.Component<{}> {
+  private plugins = [new Arrow()];
+  public render() {
+    return (
+    <div id="arrow" className="container">
+      <Flicking
+        className="flicking"
+        circular={true}
+        gap={10}
+        duration={500}
+        plugins={this.plugins}
+      >
+        <div className="panel panel0"></div>
+        <div className="panel panel1"></div>
+        <div className="panel panel2"></div>
+        <div className="panel panel3"></div>
+        <div className="panel panel4"></div>
+        <span slot="viewport" className="flicking-arrow-prev"></span>
```

Comment:
I know that there is no slot function in react. And slot attribute is used in shadow dom.

How about a slot-type function component?

```jsx
// or <ViewportSlot>
<Slot name="viewport">
    <div class="flicking-arrow-prev"></div>
</Slot>

Reply:
Okay, I'll fix it like that.